core/src/font.rs

```diff
@@ -519,14 +519,15 @@ impl GlyphShape {
 pub struct Glyph {
     // Handle to registered shape.
     // If None, it'll be loaded lazily on first render of this glyph.
-    shape_handle: RefCell<Option<ShapeHandle>>,
+    // It's a double option; the outer one is "have we registered", the inner one is option because it may not exist
+    shape_handle: RefCell<Option<Option<ShapeHandle>>>,
```

Comment:
to be sure, there currently doesn't exist a case where we put a `None` here - it's for a future PR?

Reply:
Yeah, font files can define properties of a glyph without any way to display it - they'll be whitespace characters

Reply:
Wouldn't this be nicer as an `enum`?